calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/EventManagement.java

```diff
@@ -787,434 +789,423 @@ public void inputRecurringInfoEvent(repeatType repeatType,
+       */
+      public enum priorityType {
+        None, Normal, High, Low;
+      }
 
-  /**
-   * Define a type of repeat Daily Weekly Monthly Yearly
-   */
-  public enum repeatType {
-    Daily, Weekly, Monthly, Yearly;
-  }
+      /**
+       * Available option
+       */
+      public enum selectAvailableOption {
+        AVAILABLE, BUSY, OUTSIDE
+      }
 
-  /**
-   * Define a type of repeat on MO TU WE TH FR SA SU
-   */
-  public enum repeatOn {
-    MO, TU, WE, TH, FR, SA, SU;
-  }
+      /**
+       * Define a type of repeat Daily Weekly Monthly Yearly
+       */
+      public enum repeatType {
+        Daily, Weekly, Monthly, Yearly;
+      }
 
-  /**
-   * Define a type of repeat Never After ByThisDate
-   */
-  public enum repeatEndType {
-    Never, After, ByThisDate;
-  }
+      /**
+       * Define a type of repeat on MO TU WE TH FR SA SU
+       */
+      public enum repeatOn {
+        MO, TU, WE, TH, FR, SA, SU;
+      }
 
-  /**
-   * Define a type of delete recurring Only this event Following events All events
-   */
-  public enum recurringType {
-    ONLY_EVENT, FOLLOW_EVENT, ALL_EVENT;
-  }
+      /**
+       * Define a type of repeat Never After ByThisDate
+       */
+      public enum repeatEndType {
+        Never, After, ByThisDate;
+      }
 
-  public void checkEventPopUp(){
-    switchTo().activeElement();
-    $(byText("Add Event")).should(Condition.exist);
-    assertEquals("Event title",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[2]/form/div[1]/div[2]/input")).getAttribute("placeholder"));
-    assertEquals("All",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[2]/form/div[1]/div[2]/span/select")).getText());
-    assertEquals("All day",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[2]/form/div[2]/div[2]/div/label/span")).getText());
-    assertEquals("Location",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[2]/form/div[2]/div[3]/div[1]")).getText());
-    assertEquals("Enter a location for this event",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[2]/form/div[2]/div[3]/div[2]/input")).getAttribute("placeholder"));
-    assertEquals("Participants",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[2]/form/div[2]/div[4]/div[1]")).getText());
-    assertEquals("",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[2]/form/div[2]/div[4]/div[2]/div/div/input")).getAttribute("placeholder"));
-    assertEquals("Clear",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[3]/div/button[3]")).getText());
-    assertEquals("Save",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[3]/div/button[1]")).getText());
-    assertEquals("Cancel",$(byXpath("//*[@id=\"ExoCalendarEventForm\"]/div[1]/div[3]/div/button[2]")).getText());
+      /**
+       * Define a time of reminder by email minutes, hours, days, weeks
+       */
+      public enum timeEmailReminder {
+        minutes, hours, days, weeks;
+      }
 
-  }
```

Comment:
Why inlining this method in thers methods ? Shouldn't we keep it ?

Reply:
This method contains assertions, i have added those asserts directly in the test case

Reply:
OK I see. Can't we keep this method somewhere else to avoid duplicating the code ?

Reply:
Yes done.